tools/normalize_mri_protocol_range_data.php

```diff
@@ -0,0 +1,108 @@
+<?php
```

Comment:
@ridz1208 Shouldn't this script be moved to `tools/single_use` instead of being in the `tools` directory directly?

Reply:
Moved the script in the `single_use` tool's directory in commit b174ebb.